README.md

```diff
@@ -160,6 +160,18 @@ optional arguments:
   --params PARAMS       comma separated key value pairs (key1=val1,key2=val2)
   --get-health          Get the health status of the server
   --get-identifier      Identify important information regarding the server and the system
+  --list-output-configuration-sources OUTPUT_ID
```

Comment:
Do these new options need to be added after line 91 also?

Reply:
Yes, they are missing.